diff --git a/node_modules/@redwoodjs/core/config/babel-preset.js b/node_modules/@redwoodjs/core/config/babel-preset.js
index e1c8c09..0d568f8 100644
--- a/node_modules/@redwoodjs/core/config/babel-preset.js
+++ b/node_modules/@redwoodjs/core/config/babel-preset.js
@@ -11,7 +11,7 @@ const TARGETS_NODE = '12.16'
 // Warning! Use the minor core-js version: "corejs: '3.6'", instead of "corejs: 3",
 // because we want to include the features added in the minor version.
 // https://github.com/zloirock/core-js/blob/master/README.md#babelpreset-env
-const CORE_JS_VERSION = '3.6'
+const CORE_JS_VERSION = '3.16'
 
 /** @type {import('@babel/core').TransformOptions} */
 module.exports = () => {
